SparkConnector/setup.py

```diff
@@ -36,10 +36,6 @@
 
 data_files_spec = [
     ("share/jupyter/labextensions/@swan-cern/sparkconnector", lab_path, "**"),
-    ("etc/jupyter/jupyter_server_config.d",
```

Comment:
Is this fine @krishnan-r ? (plus the addition to the `setup.py` of `SwanPortAllocator`)

Reply:
yes